plugin/fuzz/source/dextool/plugin/fuzz/backend/fuzzvariant.d

```diff
@@ -270,7 +270,7 @@ final class FuzzVisitor(RootT, ProductT) : Visitor {
             v.accept(visitor);
 
             root.put(visitor.root);
-            container.put(visitor.root, visitor.root.fullyQualifiedName);
+            //container.put(visitor.root, visitor.root.fullyQualifiedName);
```

Comment:
Yes, old technical debt that got fixed. It is up to the plugins to store the CppClass'es if they are needed